Description: Fix libesmtp linking
 Switch to use pkg-config to detect the right flags $CFLAGS and $LIBS.
Origin: vendor
Author: Salvatore Bonaccorso <carnil@debian.org>
Last-Update: 2021-10-24

--- a/configure.ac
+++ b/configure.ac
@@ -8,6 +8,7 @@ AM_INIT_AUTOMAKE(1.11 dist-bzip2)
 AM_MAINTAINER_MODE
 
 AC_PROG_CC
+PKG_PROG_PKG_CONFIG
 
 AC_HEADER_STDC
 
@@ -30,11 +31,11 @@ then
 	AC_CHECK_HEADER(libesmtp.h, ,[AC_MSG_ERROR([libesmtp.h not found in $with_libesmtp])])
 fi    
 AC_MSG_CHECKING(for libESMTP)
-if libesmtp-config --version > /dev/null 2>&1
+if $PKG_CONFIG --exists 'libesmtp-1.0'
 then
 	AC_MSG_RESULT(yes)
-	CFLAGS="$CFLAGS `libesmtp-config --cflags`"
-	LIBS="$LIBS `libesmtp-config --libs`"
+	CFLAGS="$CFLAGS `$PKG_CONFIG --cflags 'libesmtp-1.0'`"
+	LIBS="$LIBS `$PKG_CONFIG --libs 'libesmtp-1.0'`"
 else
 	AC_MSG_RESULT(no)
 	AC_MSG_ERROR(libESMTP library not found)
